Whiskey/en-US/about_Whiskey.help.txt

```diff
@@ -0,0 +1,141 @@
+TOPIC
+    about_Whiskey
+    
+SHORT DESCRIPTION
+    Whiskey is a Continuous Integration/Continuous Delivery build platform.
+    
+LONG DESCRIPTION
+        
+    ## Overview
+
+    Think of Whiskey as a build server you can run from the command line. 
+    Instead of writing a bunch of PowerShell/batch/whatever scripts, what you 
+    want to happen during your build is written as [YAML](http://yaml.org/) 
+    into a "whiskey.yml" text file and committed into the root of your 
+    source code repository.
+
+    ## System Requirements
+
+    * PowerShell 4
+    * A build server. Jenkins, TeamCity, or Appveyor are supported.
+
+    ## Getting Started
+
+    1. Create a `.whiskey` directory in the root of your source code 
+       repository.
+    2. [Download Whiskey](https://www.powershellgallery.com/packages/Whiskey)
+       from the PowerShell Gallery and put it in the ".whiskey" directory. 
+    3. Copy the ".whiskey\build.ps1" script into the root of your repository.
+    4. Copy the ".whiskey\whiskey.sample.yml" file into the root of your 
+       repository. Rename it to "whiskey.yml".
+    5. Add the .whiskey directory and the build.ps1 and whiskey.yml files to 
+       your repository. If you're using Git, run:
+           
+           > git add .whiskey build.ps1 whiskey.yml
+           > git commit -m "Adding Whiskey."
+           
+    6. Run your build:
+        
+           > .\build.ps1
+
+    ## Configuring Your Build
+
+    ### Overview
+
+    A Whiskey build has two phases: building and publishing (i.e. 
+    promoting). These are handled by the `BuildTasks` and `PublishTasks` 
+    pipelines in your whiskey.yml file. A pipeline is a list of tasks to 
+    perform when running that pipeline. Your first pipelines are empty:
+
+       BuildTasks:
+       
+       PublishTasks:
+       
+    To define a pipeline, prefix each task name with `- ` after its 

```

Comment:
This paragraph is weird, it's describing to someone **how** to write valid YAML instead of **what** to write in YAML. We should be describing what kind of configuration is expected in the YAML, i.e. Lists, Dictionaries, etc, and not proper YAML syntax.

e.g. Documentation usually reads like "Create an XML element named X nested inside of Y", not "Write the value X enclosed within a left [<] and right [>] angle bracket".

Reply:
That was on purpose. I don't want someone to have to go read a full YAML tutorial.

I've cleared up the language a bit.